plugins/foreman_remote_execution/0.1/index.md

```diff
@@ -0,0 +1,407 @@
+running CentOS 7.1 or hosts with 1 GB memory. You can check how many
+hosts such search query covers by clicking refresh button below.
+
+You can also specify **Type of query**. Currently we support two
+options
+
+- **Static query** - evaluates the search query immediately when you
+    hit the Submit button and saves the resulting set as the target of
+    this execution
+- **Dynamic query** - stores the search query and does the evaluation
+    just before the command is executed, which means if there are new
+    hosts added meanwhile, the job will be executed on them as well
+
+Dynamic query will be more powerful when we add scheduling job executions in
+future. Also it will be useful when Bookmark is used for specifying the
+targets.
+
+Then there's the template selection. If you have multiple job templates for a
+given **Job name** you'll have to select one. If there's just one like in my
+example, you'll just have to provide values for all user inputs. After
+submiting the form, you're redirected to the Job detail.
+
+## 3.3 Job detail
+
+On the Job detail page you can see overall progress and result of the job
+execution.
+
+![Job detail 1](/plugins/foreman_remote_execution/{{page.version}}/job_detail_1.png)
+
+In the overview tab you can see what query was used to find the hosts and how
+many hosts were found using it. Below is the evaluated template in generic
+form. If there were some fact or parameter inputs used, you will see only
+placeholders for them since they are specific to each host. User inputs are
+evaluated since they are same for all hosts. A chart is showing the status of
+the job on all targets.
+
+From this screen you can even cancel the job if it's not finished yet.  First
+cancel attempt will try to cancel the run on the Smart Proxy. If proxy does not
+respond, you can hit the cancel button a second time which forces the Job to be
+cancelled on Foreman (while it still might be running on Smart Proxy). Last Job
+Task takes you to Foreman Tasks that represents this Job.
+
+In Hosts tab you can see all hosts targeted by this execution, status
+of the job on them and link to Foreman Task that represent the
+execution for a particular host. Note that this task is a subtask of a
+whole Job task.
+
+
+## 3.4 Jobs list
+
+You can find all Jobs when you navigate to Monitor > Jobs. A table
+lists all jobs from history to future. You can search jobs by **Job name** with
+a queries like
+
+    job_name ~ set*
+
+This example would find all executions that run job which names starts with
+"set".
+
+![Job invocations list](/plugins/foreman_remote_execution/{{page.version}}/job_invocations.png)
+
+# 4. Advanced topics
+
+In this section we try to explain how things work behind the scenes.
+
+## 4.1 Determining the smart proxy for host
+
+In order to perform the job on a host, there needs to be a Smart Proxy
+available to do the job. In the default install, it was use all Smart proxies
```

Comment:
`s/it was/it would/`. Also noting, that this happens unless some proxy is attached through a network interface would be good